.eslintrc.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
   ignorePatterns: [
     '/libs/deps/*',
     '/tools/loc/*',
+    '/tools/floodgate/*',
```

Comment:
Why are you omitting it from linting rules? Usually this is reserved for external libraries, etc

Reply:
@meganthecoder Thanks for taking a look.

I am mostly following the model followed by Milo localization.
AFAIK, some of these configs were updated to unblock the feature from being implemented on the client side. But it is not scalable on the client-side. Once the basic functionality is working for FG and Loc, most of the related code currently in the repo will move to AIO.

@janaki-r-bhagwath who worked on Milo Loc might have additional context and reasoning behind it.

Reply:
@janaki-r-bhagwath  Was the eslint config updated to avoid the helix libs? or is there more to it?
Could this be removed for FG?

Reply:
I did not add the eslint skip for loc - usually when I do local development I ensure to enable it so all loc files are lint compliant. But yes loc has prebuilt helix libraries which are not needed in Floodgate. 
So don't think you need to skip linting in floodgate.

Reply:
@janaki-r-bhagwath @meganthecoder 
Removed the path from the lint configs. Could you guys take a look again? Thanks!

Reply:
@janaki-r-bhagwath @meganthecoder Lint errors fixed and pushed